API.yaml

```diff
@@ -659,16 +685,35 @@ paths:
           type: string
       requestBody:
         description: Query parameters to fetch the XY model.
-          Usually contains these parameters, but not limited to
-          - entries, array of all the entry (series) IDs
-          - nb, number of data points to sample
-          - startTime
-          - endTime
+          When 'requested_times' size is 2 it is the start and end of the requested range, otherwise it is the explicit array of requested sample times.
         required: true
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/Query'
+              type: object
+              properties:
+                parameters:
+                  type: object
+                  properties:
+                    requested_times:
+                      type: array
+                      items:
+                        type: integer
+                    requested_items:
+                      type: array
+                      items:
+                        type: integer
+                  required:
+                    - requested_times
+                    - requested_items
```

Comment:
requested_item should be also described in the description above

Reply:
The requested times could show more than 2 items in the example, to clearly show the difference (explicit array of requested sample times) with other places where the same name, requested times, is used for [start, end].